INITIAL-SETUP.md

```diff
@@ -0,0 +1,252 @@
+# Inital Setup
+
+Initial setup of the project will be divided into 2 main steps 
+
+* Setting up the ODK Central Server on your local machine
+* Setting up the Android app in Android Studio
+---
+## Setting up the ODK Central Server on your local machine
+
+For installing the ODK Central server locally on your machine, Make sure you are running **Docker Engine v23.x** and **Docker Compose v2.16.x** or greater.
+
+```
+docker --version && docker compose version
+```
+ <br> 
+
+## Setting Up Central
+ <br> 
+
+* Download the software.
+
+    ```
+    git clone https://github.com/getodk/central
+    ```
+* Go into the new central folder:
+    ```
+    cd central
+    ```
+* Get the latest client and server:
+    ```
+    git submodule update -i
+    ```
+* Update settings.
+    ```
+    cp .env.template .env
+    nano .env
+    ```
+* Make the following changes in `.env` file
+    * Change the ``Domain`` value to your machine's local IP address. For Eg. DOMAIN=192.168.29.147. 
```

Comment:
You can check you IP address using `ifconfig`. If you're using ethernet, it should probably be named similar to `eth*`; if you're on WLAN network, it should be `wlo*`.